src/ruby/yast/y2logger.rb

```diff
@@ -6,6 +6,41 @@
 require "yast/logger"
 
 module Yast
+
+  # class for storing the log group result data, used in the Y2Logger.group
+  class LogGroupResult
+    # @!attribute [rw] result summary, human readable description
+    # @return [String,nil] result of the step as a textual description
+    attr_accessor :summary
+
+    # @!attribute [rw] result of the block
+    # @return result of the step
+    attr_accessor :result
+
+    # @!attribute [w] set to true if the result of the group is a failure,
+    #  overrides the state evaluated from the the `result` attribute
+    attr_writer :failed
+
+    # these return values are considered failures by default,
+    # can be overridden by setting the `failed` attribute
+    FAILURES = [:abort, :cancel, false]
```

Comment:
I would make this configurable. So you can specify what is failure

Reply:
good. Explicit one is ok...BTW I am missing test for explicit failed and I also failed to find it in code. I only see `group.success =`

Reply:
That's because Lada has changed `failed` to `success` in the meantime, on my suggestion :)